packages/eslint-plugin/src/rules/sort-ngmodule-arrays.ts

```diff
@@ -0,0 +1,69 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { MODULE_CLASS_DECORATOR } from '../utils/selectors';
+import {
+  getDecoratorPropertyValue,
+  isArrayExpression,
+  isIdentifier,
+} from '../utils/utils';
+
+import { NgModule } from '@angular/compiler/src/core';
```

Comment:
Please convert this to a type only import, we do not want the angular compiler to be bundled with the eslint-plugin